docs/installation.md

```diff
@@ -295,6 +259,35 @@ Go to the `conda-forge`
 for more details.
 
 
+## JAX nightly installation
+
+Nightly releases reflect the state of the main JAX repository at the time they are
+built, and may not pass the full test suite.
+
+- CPU only:
+
+```bash
+pip install -U --pre jax -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html
+```
+
+- Google Cloud TPU:
+
+```bash
+pip install -U --pre jax[tpu] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+```
+
+- NVIDIA GPU (CUDA 12):
+
+```bash
+pip install -U --pre jax[cuda12]] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html
+```
+
+- NVIDIA GPU (CUDA 12) legacy:
```

Comment:
It may not be clear what "legacy" means here. We should add a sentence on why someone might want to use this version.